in_text/text_04_predictions.yml

```diff
@@ -76,800 +76,1124 @@ cross-cites:
-    -
-      attr-label: ELEV_MASL
-      attr-def: >-
-        Elevation of reservoir surface in meters above sea level.
-      attr-defs: https://doi.org/10.1890/100125
-      data-min: 4
-      data-max: 419
-      data-units: meters
+        Predicted water temperature
+      attr-defs: This data release
+      data-min: -0.196
+      data-max: 23.461
+      data-units: degrees Celsius
     -
-      attr-label: CATCH_SKM
-      attr-def: >-
-        Area of upstream catchment draining into the reservoir.
-      attr-defs: https://doi.org/10.1890/100125
-      data-min: 44
-      data-max: 1177
-      data-units: square kilometers
+      attr-label: flow
+      attr-def:  >-
+        Predicted outflow
+      attr-defs: This data release
+      data-min: 0.131
+      data-max: 762.163
+      data-units: cubic meters per second
   -
-    data-name: >-
-      sntemp_inputs_outputs_{subset}.zip
+    data-name: dwallin_stream_preds.csv.zip
     data-description: >-
-      PRMS-SNTemp inputs and outputs, including weather driver data, intermediate varibles, and
-      model predictions. There are two files; files either contain inputs/outputs from all 456 stream reaches in the
-      Delaware River Basin modeling network, or for the 56 segments above Lordville, New York.
-
-      
-      Code used to generate these inputs and outputs can be found in the repository "run-prms-sntemp" in Zwart and others 2021 (https://doi.org/10.5281/zenodo.5164910).
-
-      
-      There are two error codes in PRMS-SNTemp v5.1.0_rc that can be present in the
-      seg_tave_water and seg_tave_upstream attributes indicating no streamflow for the current
-      timestep (error code: -98.9) or no streamflow at any timestep (error code: -99.9) of the model run.
-      Stream segments that never have streamflow (-99.9) do not have any hydrologic reach units and do not have any upstream segments to produce
-      streamflow, and consequently never be able to simulate stream temperature. Segments with
-      contributing hydrologic reach units and/or upstream segments can have streamflow but
-      may not have streamflow at the current timestep (-98.9)
-
-      
-      Very negative groundwater and shallow subsurface water temperatures (e.g. less than -1 C) are
-      not error codes, but rather are a product of the way this version of PRMS-SNTemp is coded. However,
-      we advise against using these very low groundwater and shallow subsurface temperature values
-      as they likely are far from actual water temperatures.
+      Stream temperature predictions from the distance-weighted-average lotic-lentic input network (DWALLIN).
+      Predictions were generated from a combination of lotic (from PRMS-SNTemp) and lentic (GLM) models.
+      See dwallin_functions.R for processing steps that were taken to calculate the DWALLIN stream temperature
+      predictions.
+  
+      There a few site-dates with extremely high predicted stream temperature values.
```

Comment:
This detail related to dwallin_stream_preds.csv will need to be updated @limnoliver.